discovery-provider/docker-compose.acdc.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.9'
+services:
+  acdc-test:
+    image: acdc-test
+    restart: unless-stopped
+    healthcheck:
+      test:
+        [
+          "CMD-SHELL",
+          "curl --fail localhost:8545/health || exit 1"
+        ]
+      interval: 5s
+      timeout: 5s
+      retries: 12
```

Comment:
Maybe it makes sense to make this file / image its own repo that we push to dockerhub with? i'm not sure it makes sense in the discprov directory.